### PR TITLE
adding the missing /etc:/host/etc:ro for modern-bpf

### PR DESCRIPTION
**Pull request by ycaoT**
**What type of PR is this?**
Update the website to add missing options, to solve issues for users to run falco agent in containers. 
> Uncomment one (or more) `/kind <>` lines:
> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

> /kind content

> /kind translation

> /kind event

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

> /area documentation

> /area community

**What this PR does / why we need it**:
https://github.com/falcosecurity/falco/issues/2990#issuecomment-1883470260

**Which issue(s) this PR fixes**:
https://github.com/falcosecurity/falco/issues/2990

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by incertum — APPROVED** [review]
LGTM, but I can't officially approve!

CC @Andreagit97 since you planned to rework the Install guides.

File: content/en/docs/install-operate/running/index.md [hunk 1]
```diff
@@ -272,6 +272,7 @@ docker run --rm -i -t \
            --privileged \
            -v /var/run/docker.sock:/host/var/run/docker.sock \
            -v /proc:/host/proc:ro \
+           -v /etc:/host/etc:ro \
            falcosecurity/falco-no-driver:latest falco --modern-bpf
 ```
```
Comment (Andreagit97) [inline content/en/docs/install-operate/running/index.md:278]: * `-v /var/run/docker.sock:/host/var/run/docker.sock`  needed to scrape docker container info and extract fields like `container_name`. Please note that if the container is spawned with another container runtime like `containerd` you need to mount the corresponding socket `-v /var/run/containerd/containerd.sock:/host/var/run/containerd/containerd.sock` and provide Falco with the corresponding command line flag `--cri /var/run/containerd/containerd.sock`
* `-v /proc:/host/proc:ro` needed to read the list of processes running on the host.
* `-v /etc:/host/etc:ro` needed to resolve users on the host.

**Review by Andreagit97 — COMMENTED** [review]
Great catch! thank you for this!
@mikegcoleman Maybe in the new doc refactor, we want to add a more general command that catches also containers spawned with `containerd` and not just by docker (the actual docker command just tracks containers spawned by docker).

This is a more complete command with also the containerd support

```bash
docker run --rm -i -t \
           --privileged \
           -v /var/run/docker.sock:/host/var/run/docker.sock \
           -v /var/run/containerd/containerd.sock:/host/var/run/containerd/containerd.sock \
           -v /proc:/host/proc:ro \
           -v /etc:/host/etc:ro \
           falcosecurity/falco-no-driver:latest falco --modern-bpf --cri /var/run/containerd/containerd.sock
```

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**